crates/openapi/src/routes/payments.rs

```diff
@@ -630,6 +630,25 @@ pub fn payments_post_session_tokens() {}
 #[cfg(feature = "v2")]
 pub fn payments_create_intent() {}
 
+/// Payments - Get Intent
+///
+/// **Get a payment intent object when id is passed in path**
+///
+/// You will require the 'API - Key' from the Hyperswitch dashboard to make the call.
+#[utoipa::path(
+  get,
+  path = "/v2/payments/{id}/get-intent",
+  params (("id" = String, Path, description = "The unique identifier for the Payment Intent")),
```

Comment:
will we have merchant_id and profile_id in the headers?

Reply:
Will create a separate PR for that